Sources/AppStoreConnectCLI/Services/AppStoreConnectService.swift

```diff
@@ -26,13 +26,30 @@ class AppStoreConnectService {
     }
 
     func readCertificate(serial: String) throws -> Certificate {
-        try ReadCertificateOperation(options: .init(serial: serial)).execute(with: requestor).await()
+        let sdkCertificate = try ReadCertificateOperation(options: .init(serial: serial))
+            .execute(with: requestor)
+            .await()
+
+        return Certificate(sdkCertificate)
     }
 
     func createCertificate(with options: CreateCertificateOptions) -> AnyPublisher<Certificate, Error> {
         CreateCertificateOperation(options: options).execute(with: requestor)
     }
 
+    func revokeCertificates(serials: [String]) throws {
+        let certificatesIds = try serials.map {
+            try ReadCertificateOperation(options: .init(serial: $0))
+                .execute(with: requestor)
+                .await()
+                .id
+            }
```

Comment:
The closing brace should match the indentation level of the line that has the opening brace
```suggestion
        }
```